### PR TITLE
grap 1.45

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/grap.rb [hunk 1]
```diff
@@ -1,9 +1,7 @@
...
-
 class Grap < Formula
   homepage "http://www.lunabase.org/~faber/Vault/software/grap/"
-  url "http://www.lunabase.org/~faber/Vault/software/grap/grap-1.44.tar.gz"
-  sha1 "b15e4e04525301cfbd2900b1f1a028e81a12ba92"
+  url "http://www.lunabase.org/~faber/Vault/software/grap/grap-1.45.tar.gz"
+  sha256 "906743cdccd029eee88a4a81718f9d0777149a3dc548672b3ef0ceaaf36a4ae0"
 
   def install
     system "./configure", "--disable-debug",
```
Comment (mikemcquaid) [inline Library/Formula/grap.rb]: Looks like --disable-debug isn't recognised: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24779/version=mavericks/testReport/junit/brew-test-bot/mavericks/install_grap/

**Comment by mikemcquaid** [comment]
failing on mavericks: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24779/version=mavericks/testReport/junit/brew-test-bot/mavericks/install_grap/

**Comment by dunn** [comment]
Not sure what that error means—any ideas?

**Comment by mikemcquaid** [comment]
Looks like `make` is parallel-safe but `make install` may not be. Splitting it into two steps and adding a `ENV.deparallelize` before `make install` should do it.

File: Library/Formula/grap.rb [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+    system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}",
                           "--with-example-dir=#{share}/grap/examples"
+    system "make"
+    ENV.deparallelize
     system "make", "install"
   end
+
+  test do
```
Comment (asparagui) [inline Library/Formula/grap.rb]: :smiley_cat:

**Comment by asparagui** [comment]
Thanks for this and all your other pull requests!

**Pull request closed without merging**